### ISSUE TITLE
`Picture` element images: Missing alt text

### ISSUE THREAD

**Issue by mukeshpanchal27** [WordPress/performance#1315]
## Bug Description

When using the new Modern Image Formats `picture` element support, I noticed that the markup didn't add the alt text that supply by the editor.

## Steps to reproduce

1. Activate the Modern Image Formats plugin and enable `picture` support from the Media Settings screen.
2. Add an image block and add alt text in Block settings > ALTERNATIVE TEXT
3. Preview the page in the front end

### Editor
<img width="996" alt="Screenshot 2024-06-24 at 4 54 56 PM" src="https://github.com/WordPress/performance/assets/10103365/114d8bd5-3f55-4dc3-bd71-6394b819fcf1">

### Frontend
<img width="1332" alt="Screenshot 2024-06-24 at 4 55 08 PM" src="https://github.com/WordPress/performance/assets/10103365/ba7c3310-dee6-4194-a02f-d10daa00aaae">

**Comment by devansh016** [comment]
@mukeshpanchal27 @westonruter 
Can you please assign me this issue. Would like to work on it.
